src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.macOS.cs

```diff
@@ -0,0 +1,15 @@
+﻿#if !__ANDROID__ && !__IOS__ && !(WINDOWS || WINDOWS_UWP) && !UNO_REFERENCE_API
+using System.Threading.Tasks;
+
+namespace Uno.Toolkit.UI;
+
+public partial class ExtendedSplashScreen
+{
+	private static Task<FrameworkElement?> GetNativeSplashScreen(SplashScreen? splashScreen)
```

Comment:
```suggestion
	private static Task<FrameworkElement?> GetNativeSplashScreen()
```
Update for post #1061